index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+        <title>ShEx-lite Evolution</title>
+        <link rel="shortcut icon" type="image/x-icon" href="https://drive.google.com/uc?export=view&id=1DCOKHnBI-XqERTdVphfbtTlTiAukO0rX" />
```

Comment:
I guess this link is the external asset for the weso logo image isnt it? Some comments sometimes help...